### PR TITLE
improve default conf and README

### PR DESCRIPTION
**Pull request by anirudhmendiratta**
1. Add example for running server in docker and connecting aql to it
2. Improve template conf (either comment out variables like `${SERVICE_ADDRESS}` or replace them with default values when they are necessary, e.g. port). Annoyingly, this is necessary to be able to parse the conf at startup without error.

### CODE REVIEW

File: aerospike.template.conf [hunk 1]
```diff
@@ -6,16 +6,17 @@ service {
...
+	# transaction-queues ${TRANSACTION_QUEUES}
+	# transaction-threads-per-queue ${TRANSACTION_THREADS_PER_QUEUE}
 	proto-fd-max 15000
 }
 
 logging {
 
 	# Log file must be an absolute path.
-	file ${LOGFILE} {
+	file /var/log/aerospike/aerospike.log {
```
Comment (tevren) [inline aerospike.template.conf]: Why make this static?

Comment (anirudhmendiratta) [reply aerospike.template.conf]: Annoyingly, the configuration file does not pick these up from the environment. There are 2 mutually exclusive ways to configure these parameters, i) use environment variables or ii) hardcode values in the conf file. The conf file is recommended but the default version of this file cannot be parsed as is, and thus I added some values for what was necessary and commented out the rest of the params (default values will be used for those)

File: README.md [hunk 2]
```diff
@@ -69,6 +69,54 @@ A full example:
...
+```
+OR
+
+ii) Use `aerospike.example.conf` as a configuration file:
+```
+docker run -ti -v <absolute_path_to_local_repo>:/opt/aerospike/etc --name aerospike -p 3000:3000 -p 3001:3001 -p 3002:3002 -p 3003:3003 aerospike/aerospike-server /usr/bin/asd --foreground --config-file /opt/aerospike/etc/aerospike.example.conf
+```
+
+**Running a client cli that talks to this server**:
+1. Start `aql` (stands for aerospike query language, this is analogous to `psql` or `redis-cli`):
```
Comment (tevren) [inline README.md:90]: Thank you for adding the following sections on how to use the server/client. It was very helpful in reviewing this PR.

**Review by tevren — APPROVED** [review]
Looks good to me!

**Pull request merged**